project/Dependencies.scala

```diff
@@ -184,6 +186,43 @@ object Dependencies {
       "com.typesafe.play" %% "play-ahc-ws-standalone" % PlayStandaloneWsVersion,
       "com.typesafe.play" % "shaded-asynchttpclient" % PlayStandaloneWsVersion,
       "com.typesafe.play" % "shaded-oauth" % PlayStandaloneWsVersion,
+      playTest,
+      // dependencies added by play-test
+      "org.fluentlenium" % "fluentlenium-core" % "3.5.2",
+      "org.seleniumhq.selenium" % "selenium-support" % "3.9.1",
+      "org.seleniumhq.selenium" % "selenium-api" % "3.9.1",
+      "org.seleniumhq.selenium" % "selenium-remote-driver" % "3.9.1",
+      "org.seleniumhq.selenium" % "selenium-firefox-driver" % "3.9.1",
+      "net.bytebuddy" % "byte-buddy" % "1.7.9",
+      "org.apache.commons" % "commons-exec" % "1.3",
+      "commons-logging" % "commons-logging" % "1.2",
+      "com.google.code.gson" % "gson" % "2.8.2",
+      "com.squareup.okhttp3" % "okhttp" % "3.9.1",
+      "com.squareup.okio" % "okio" % "1.13.0",
+      "org.atteo.classindex" % "classindex" % "3.4",
+      "org.seleniumhq.selenium" % "htmlunit-driver" % "2.31.0",
+      "net.sourceforge.htmlunit" % "htmlunit" % "2.31",
+      "xalan" % "xalan" % "2.7.2",
+      "xalan" % "serializer" % "2.7.2",
+      "org.apache.commons" % "commons-text" % "1.3",
+      "org.apache.httpcomponents" % "httpmime" % "4.5.5",
+      "org.apache.httpcomponents" % "httpclient" % "4.5.5",
+      "org.apache.httpcomponents" % "httpcore" % "4.4.9",
+      "net.sourceforge.htmlunit" % "htmlunit-core-js" % "2.31",
+      "net.sourceforge.htmlunit" % "neko-htmlunit" % "2.31",
+      "xerces" % "xercesImpl" % "2.11.0",
+      "xml-apis" % "xml-apis" % "1.4.01",
+      "net.sourceforge.htmlunit" % "htmlunit-cssparser" % "1.0.0",
+      "commons-io" % "commons-io" % "2.6",
+      "commons-net" % "commons-net" % "3.6",
+      "org.eclipse.jetty.websocket" % "websocket-client" % "9.4.10.v20180503",
+      "org.eclipse.jetty" % "jetty-client" % "9.4.10.v20180503",
+      "org.eclipse.jetty" % "jetty-http" % "9.4.10.v20180503",
+      "org.eclipse.jetty" % "jetty-util" % "9.4.10.v20180503",
+      "org.eclipse.jetty" % "jetty-io" % "9.4.10.v20180503",
+      "org.eclipse.jetty" % "jetty-xml" % "9.4.10.v20180503",
+      "org.eclipse.jetty.websocket" % "websocket-common" % "9.4.10.v20180503",
+      "org.eclipse.jetty.websocket" % "websocket-api" % "9.4.10.v20180503",
```

Comment:
That's huge! I added a dependency on play-test and had to add all that to the whitelist. :-(

Reply:
Whoa quite a bit. Why do we need to list this explicitly? Just for my understanding

Reply:
I have the same question. 

I guess the whitelist check could be improved and not check transitive dependencies from a library added on `Test` scope.

Reply:
Super weird... not sure I get it still... :\ Maybe some day

Reply:
@ktoso let me know if you'd like some more context